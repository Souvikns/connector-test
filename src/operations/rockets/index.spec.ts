import {Route} from 'OpenApiRouter'
import {handleRockets} from '.'

describe("Rockets Operation", () => {
    it("should return a 200 response", async() => {
        const route: Route = {
            route: "/rockets/{rocket-id}",
            operation: {
                "operationId": "rockets",
                responses: {}
            },
            pathParameters: {
                "rocketId": "5e9d0d95eda69974db09d1ed"
            }
        }
        const response = await handleRockets(route)

        expect(response).toMatchSnapshot(`Promise {}`);
    })
})