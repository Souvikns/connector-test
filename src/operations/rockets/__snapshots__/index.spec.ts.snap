// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets Operation should return a 200 response: Promise {} 1`] = `
Object {
  "body": Object {
    "company": "SpaceX",
    "cost_per_launch": Object {
      "amount": 90000000,
      "currency": "USD",
    },
    "country": "United States",
    "id": "5e9d0d95eda69974db09d1ed",
    "main_image": "https://farm5.staticflickr.com/4599/38583829295_581f34dd84_b.jpg",
  },
  "status": 200,
}
`;
